src/System.CommandLine/SymbolExtensions.cs

```diff
@@ -15,14 +16,14 @@ internal static class SymbolExtensions
 
         internal static bool ShouldShowHelp(this ISymbol symbol) =>
             !symbol.IsHidden &&
-            (!string.IsNullOrWhiteSpace(symbol.Name) ||
-             !string.IsNullOrWhiteSpace(symbol.Description) ||
+            (!String.IsNullOrWhiteSpace(symbol.Name) ||
+             !String.IsNullOrWhiteSpace(symbol.Description) ||
              symbol.Arguments().Any(a => a.ShouldShowHelp()));
 
         internal static bool ShouldShowHelp(
             this IArgument argument) =>
             argument != null &&
-            !string.IsNullOrWhiteSpace(argument.Name) &&
+            !String.IsNullOrWhiteSpace(argument.Name) &&
```

Comment:
Is there a reason for converting away from the keyword?

Reply:
Must be a format setting. This was unintentional.